eclair-core/src/main/resources/reference.conf

```
@@ -77,13 +79,23 @@ eclair {
   fee-base-msat = 1000
   fee-proportional-millionths = 100 // fee charged per transferred satoshi in millionths of a satoshi (100 = 0.01%)
 
-  // maximum local vs remote feerate mismatch; 1.0 means 100%
-  // actual check is abs((local feerate - remote fee rate) / (local fee rate + remote fee rate)/2) > fee rate mismatch
-  max-feerate-mismatch = 1.56 // will allow remote fee rates up to 8x bigger or smaller than our local fee rate
+  on-chain-fees {
+    // number of blocks to target when computing fees for each transaction type
+    fee-targets {
+      funding = 6        // target for the funding transaction
+      commitment = 2     // target for the commitment transaction (used in force-close scenario) *do not change this unless you know what you are doing*
+      mutual-close = 12  // target for the mutual close transaction
+      claim-main = 12    // target for the claim main transaction (tx that spends main channel output back to wallet)
+    }
+
+    // maximum local vs remote feerate mismatch; 1.0 means 100%
+    // actual check is abs((local feerate - remote fee rate) / (local fee rate + remote fee rate)/2) > fee rate mismatch
+    max-feerate-mismatch = 1.56 // will allow remote fee rates up to 8x bigger or smaller than our local fee rate
 
-  // funder will send an UpdateFee message if the difference between current commitment fee and actual current network fee is greater
-  // than this ratio.
-  update-fee_min-diff-ratio = 0.1
+    // funder will send an UpdateFee message if the difference between current commitment fee and actual current network fee is greater
+    // than this ratio.
+    update-fee_min-diff-ratio = 0.1
```

Ew, didn't notice this ugly underscore before!

```suggestion
    update-fee-min-diff-ratio = 0.1
```